### DISCUSSION TITLE
app_appchannel_test won't run

### DISCUSSION THREAD

**Discussion by Pandagineer** [bitcraze/discussions#255] [Q&A]
Hello. I am trying to run "app_appchannel_test" in the crazyflie-firmware "example" folder. I believe there were no issues building or flashing. The console shows "Waiting for activation ...", as expected from appchannel_test.c. But when I type "python 3 tools/appchannelTest.py", I get the following message:
= = = = = = =
bitcraze@ubuntu:~/projects/crazyflie-firmware/examples/app_appchannel_test$ python3 tools/appchannelTest.py
Scanning interfaces for Crazyflies...
[Errno 16] Resource busy
Crazyflies found:
No Crazyflies found, cannot run example
= = = = = = = 

Please advise. Thank you.

**Comment by knmcguire** [comment]
Hi!

There is an error saying
`[Errno 16] Resource busy`

which means that the crazyradio is unavailable and doing something else. 

Were you connected with the cfclient to the crazyflie? Then you need to disconnect it first before running the script

**Reply by Pandagineer** [reply]
Solved. Thanks!

**Reply by knmcguire** [reply]
Was my suggestion what solved it for you? If so, you can mark it as an answer or put your solution  in 'suggest an answer' underneath here
